libr/core/cmd.c

```diff
@@ -5044,7 +5052,14 @@ DEFINE_HANDLE_TS_FCN(arged_command) {
 }
 
 DEFINE_HANDLE_TS_FCN(legacy_quoted_command) {
-	return run_cmd_depth (state->core, node_string) != -1;
+	int res = run_cmd_depth (state->core, node_string);
```

Comment:
maybe this could just be `return run_cmd_depth (state->core, node_string)` as the values are anyway the same.

Reply:
actually no... :D